include/openssl/types.h

```diff
@@ -7,8 +7,9 @@
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef HEADER_OPENSSL_TYPES_H
```

Comment:
Doesn't this break an application that does `#include <openssl/ossl_typ.h>`?

Reply:
Oh yes, of course it does! Thanks for pointing it out.

At some time in the past, I did have a commit adding a `<openssl/ossl_typ.h>` compatibility header which forwards those includes to `<openssl/types.h>`.  It was placed in a _separate_ commit on purpose  in order not to break git's automatic rename detection for `openssl/{ossl_typ.h => types.h}`. Somehow with all my experiments, the commit got lost. I'll add it back.

Reply:
Found it in  #9274: commit  470c37f9d3dd2ef73aee3a6138e0303c4c9c1442 was probably dropped accidentally when migrating.